src/data/data.js

```diff
@@ -194,24 +194,24 @@ const pageData = {
         members: [
           {
             image: {
-              src: '',
-              alt: '',
+              src: 'https://place-hold.it/220x220/#d8d8d',
+              alt: 'foto',
```

Comment:
Hey @juan-ortizc since this is the text alternative if the user is not able to see the photo, it should describe what is the photo about, for example something like "Foto de Gloria Correa Silva"

Reply:
Done